### PR TITLE
feat: Fraction class

### PR DESCRIPTION
**Pull request by RogerMo01**
This is the Fraction class

### CODE REVIEW

File: OOP-Exercises/Fraction.cs [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+        public static Fraction Subtract(Fraction fraction1, Fraction fraction2)
+        {
+            Fraction result = new Fraction();
+            result.Denominator = MathTools.MCM(fraction1.Denominator, fraction2.Denominator);
+            result.Numerator = (result.Denominator / fraction1.Denominator) * fraction1.Numerator - (result.Denominator / fraction2.Denominator) * fraction2.Numerator;
+
+            return result;
+        }
+
+        public static Fraction Multiply(Fraction fraction1, Fraction fraction2)
```
Comment (LongJohnSilver1504) [inline OOP-Exercises/Fraction.cs:64]: This method must be private because we use the `/` operator for that

File: OOP-Exercises/Fraction.cs [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+        {
+            Fraction result = new Fraction
+            {
+                Numerator = fraction1.Numerator * fraction2.Numerator,
+                Denominator = fraction1.Denominator * fraction2.Denominator
+            };
+
+            return result.Simplify();
+        }
+        public static Fraction Divide(Fraction fraction1, Fraction fraction2)
```
Comment (LongJohnSilver1504) [inline OOP-Exercises/Fraction.cs:74]: same comment

File: OOP-Exercises/Fraction.cs (hunk 1 above)
Comment (RogerMo01) [reply OOP-Exercises/Fraction.cs:64]: Ok. Done!

File: OOP-Exercises/Fraction.cs (hunk 2 above)
Comment (RogerMo01) [reply OOP-Exercises/Fraction.cs:74]: Done!

**Pull request merged**